### PR TITLE
Openshift ci preparations

### PR DESCRIPTION
**Pull request by vatsalparekh**
**What this PR does / why we need it**:
Preparations for having OpenShift CI here replacing the broken travis
This is needed for https://github.com/openshift/release/pull/21718
**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
+COPY internal/ internal/
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -ldflags="-X 'kubevirt.io/ssp-operator/internal/template-validator/version.COMPONENT=$COMPONENT'\
+-X 'kubevirt.io/ssp-operator/internal/template-validator/version.VERSION=$VERSION'\
+-X 'kubevirt.io/ssp-operator/internal/template-validator/version.BRANCH=$BRANCH'\
+-X 'kubevirt.io/ssp-operator/internal/template-validator/version.REVISION=$REVISION'" -o cmd/kubevirt-template-validator/kubevirt-template-validator cmd/kubevirt-template-validator/main.go
+
 FROM registry.access.redhat.com/ubi8/ubi-minimal
 
 MAINTAINER "Francesco Romani" <fromani@redhat.com>
```
Comment (ksimon1) [inline Dockerfile:29]: can you please change this to `MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>`

File: hack/env/common-templates-v0.6.0.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Version v0.6.0
+# Version v0.13.1
```
Comment (ksimon1) [inline hack/env/common-templates-v0.6.0.yaml:1]: Can you please rename the file to `hack/env/common-templates-v0.13.1.yaml `

File: Dockerfile (hunk 1 above)
Comment (vatsalparekh) [reply Dockerfile:29]: Changed

**Pull request merged**